docs/intro/from-ground-up.rst

```diff
@@ -178,32 +187,40 @@ The same, but using web-poet
 
     # but after we got the response, the usage is the same
     book_page = BookPage(response=response)
-    item = book_page.extract_book()
+    item = book_page.to_item()
 
 
 Differences from a previous example:
 
-* instead of dicts with "url" and "text" fields, :class:`~.HttpResponse`
-  instances are used. :class:`~.HttpResponse` is a structure 
-  defined by web-poet acting as a generic data container for HTTP Responses.
-  *(check out the API reference of* :class:`~.HttpResponse` *for more info
-  about the fields it holds)*
+* web-poet provides a standard :class:`~.HttpResponse` class, with helper
+  methods like :meth:`~.HttpResponse.css`.
 
   Note how headers are passed when creating :class:`~.HttpResponse` instance.
   This is needed to decode body (which is ``bytes``) to unicode properly,
   using the web browser rules. It involves checking ``Content-Encoding``
   header, meta tags in HTML, BOM markers in the body, etc.
 
-* instead of ``extract_book`` function we got ``BookPage`` class,
-  which receives response data in its ``__init__`` method - see how it
-  is created: ``BookPage(response=response)``.
-* ``BookPage`` inherits from :class:`~.WebPage` base class. This base class
-  is not doing much: it
+* instead of ``extract_book`` function we've got a ``BookPage`` class,
+  which inherits from :class:`~.ItemPage` base class, receives response
+  data in its ``__init__`` method, and returns the extracted item
+  in the ``to_item()`` method.
```

Comment:
```suggestion
* instead of the ``extract_book`` function we've got a ``BookPage`` class,
  which inherits from the :class:`~.ItemPage` base class, receives response
  data in its ``__init__`` method, and returns the extracted item
  in the ``to_item()`` method.
```